From 3f35614079f9ce795a0b449dcdf292157327a0e3 Mon Sep 17 00:00:00 2001
From: Alex Wilson <alex.wilson@joyent.com>
Date: Tue, 17 Apr 2018 15:48:27 -0700
Subject: [PATCH] joyent/node-smartdc-auth#14 Agent errors result in useless
 exception Reviewed by: Cody Peter Mello <cody.mello@joyent.com>

---
 lib/kr-agent.js | 1 -
 package.json    | 2 +-
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/lib/kr-agent.js b/lib/kr-agent.js
index 23442e7..53cef6e 100644
--- a/lib/kr-agent.js
+++ b/lib/kr-agent.js
@@ -135,7 +135,6 @@ AgentKeyPair.prototype.createSign = function (opts) {
 
 		self.akp_agent.sign(pub, data, function (err, sig) {
 			if (err) {
-				cache.put(ck, err);
 				cb(err);
 				return;
 			}
diff --git a/package.json b/package.json
index 6f3113c..d1db3fb 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "smartdc-auth",
     "description": "Joyent Authentication Library",
-    "version": "2.5.6",
+    "version": "2.5.7",
     "author": "Joyent (joyent.com)",
     "contributors": [
         "Mark Cavage",
-- 
2.21.0

